align/pnr/checker.py

```diff
@@ -15,9 +15,9 @@ def check_placement(placement_verilog_d, scale_factor):
     non_leaves = {module['concrete_name'] for module in placement_verilog_d['modules']}
 
     for module in placement_verilog_d['modules']:
-        if len(module['constraints']) == 0:
-            continue  # No constraints
         constraints = module['constraints']
+        if len(constraints) == 0:
+            continue
```

Comment:
We may not want to do this. Even if there are no constraints, the checker does check to make sure bboxes do not overlap.

Reply:
That was already there. I just rewrote it a little. Should I get rid of it and try?

Reply:
Looks like I was the one who originally added this back when we weren't checking for overlaps.

Please go ahead and remove it @stevenmburns.